src/main/java/de/tum/cit/aet/artemis/lecture/domain/LectureUnit.java

```diff
@@ -116,13 +111,8 @@ public void setCompetencyLinks(Set<CompetencyLectureUnitLink> competencyLinks) {
         this.competencyLinks = competencyLinks;
     }
 
-    public LectureTranscription getLectureTranscription() {
-        return lectureTranscription;
-    }
-
-    public void setLectureTranscription(LectureTranscription lectureTranscription) {
-        this.lectureTranscription = lectureTranscription;
-    }
+    // NOTE: we explicitly do not add LectureTranscription here to avoid Hibernate loading issues because of its OneToOne relationship with is automatically EAGER and cannot be set
```

Comment:
```suggestion
    // NOTE: we explicitly do not add LectureTranscription here to avoid Hibernate loading issues because of its OneToOne relationship which is automatically EAGER and cannot be set
```